environments/variable.tf

```diff
@@ -15,3 +15,5 @@ variable "vnet_name" {}
 variable "vnet_rg" {}
 variable "product" {}
 variable "builtFrom" {}
+variable "traffic_manager_endpoints" { default = {} }
```

Comment:
is this formatted properly?

Reply:
Yes, I believe so. The module is now working as expected.